docs/ops/sort/NonMaxSuppression_5.md

```diff
@@ -0,0 +1,116 @@
+## NonMaxSuppression<a name="NonMaxSuppression"></a>
```

Comment:
Add a reference `{#openvino_docs_ops_sort_NonMaxSuppression_4}`. Refer to the specification of the version 4 for the example

Reply:
Maybe `{#openvino_docs_ops_sort_NonMaxSuppression_5}`?

Reply:
Yes, correct version is 5.

Reply:
Done.